apps/web/app/layout.tsx

```diff
@@ -1,22 +1,21 @@
-import "./globals.css";
-import type { Metadata } from "next";
-import { Inter } from "next/font/google";
+import type React from 'react';
```

Comment:
try not to import direct React from 'react', this is just our team's convention:
```typescript
import type { ReactNode } from 'react';
```